docs/user-guide/binned-data/histogramming-grouping-and-binning.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
     "## Overview\n",
     "\n",
-    "Histogramming (see [sc.hist](../../generated/functions/scipp.hist.rst#scipp.hist)), grouping (using [sc.groupby](../groupby.ipynb)), and binning (see [Binned data](binned-data.ipynb)) all serve similar but slightly different purposes.\n",
+    "Histogramming (see [sc.hist](../../generated/functions/scipp.hist.rst#scipp.hist)), grouping (using [sc.group](../../generated/functions/scipp.group.html#scipp-group)), and binning (see [Binned data](binned-data.ipynb)) all serve similar but slightly different purposes.\n",
```

Comment:
`group` and `groupby` are not the same. `group` is a variant of `bin`. Also need to undo a bunch of changes below.